core-services/release-controller/README.md

```diff
@@ -67,4 +71,6 @@ See `rpms-ocp-*.yaml` for the definition of each proxy service, `ocp-*.repo` for
 
 In general, new RPMs should be added to the correct Brew tags at Red Hat for OCP first (since we need those to eventualyl build). Those are mirrored nightly by the SRE team to `https://mirror.openshift.com/enterprise/reposync/4.Y`.  If a set of RPMs need to be added (a separate brew tag) that is compatible with the existing RPM sets, then we would add it to the appropriate `ocp-4.Y-default.repo` (which all images can use). If the repo is incompatible with any of the RPMs in the default repos (like the OpenStack project), a new type of `ocp-4.Y-NAME.repo` has to be created (similar to `ocp-4.2-openstack.repo`) that points to the correct mirrors. Then the images that want to use that as a base have to inject the correct RPM repo (TODO: future changes will mount these automatically) and then depend on that base. For an example, see the `kuryr` ci-operator definitions.
 
-See the OWNERS file in this directory for approvers.
\ No newline at end of file
+See the OWNERS file in this directory for approvers.
+
+[testgrid-config-generator]: https://github.com/openshift/ci-tools/tree/master/cmd/testgrid-config-generator
```

Comment:
this seems to be dangling, should it be up earlier in the doc where you mention updating testgrid?

Reply:
> this seems to be dangling, should it be up earlier in the doc where you mention updating testgrid?

Doesn't matter where it goes ([rendered page][1]).  But I can shift it up to be inline, or just after the paragraph that consumes it, if you prefer a different style.

[1]: https://github.com/openshift/release/blob/1e14c589f63bfb10562dbeabd65f84aa55d6a5c2/core-services/release-controller/README.md#release-informers

Reply:
it's fine, i didn't realize what this was doing.